### ISSUE TITLE
How to create an account

### ISSUE THREAD

**Issue by YaBoiMax** [jackyzy823/fxa-selfhosting#14]
I can't really create an account since I don't understand some general things

In the section
```
Check sigincode

                docker-compose logs fxa-auth-local-mail-helper |grep -i code
```
`docker-compose logs fxa-auth-local-mail-helper |grep -i code`
returns nothing

and with
```
Or (Assume your account example@test.local)

                Get Code: curl http://127.0.0.1:9001/mail/example
                Clean up: curl -X DELETE http://127.0.0.1:9001/mail/example
```
I don't understand what you mean with
`Or (Assume your account example@test.local)`

and even if I run
`curl http://127.0.0.1:9001/mail/example`
it's just stuck loading

and I can't find any noticable errors in any logs anywhere so I'm just stuck at what to do next. Any help is much appreciated

**Comment by jackyzy823** [comment]
Have you  completed the sign-up process in the content server ( https://<content>.your.domain just like https:/accounts.firefox.com) ?

1. You can directly visit that site
2. You can click firefox's menu "sign in" right to "sync and save data" if you have properly configured your firefox with custom fxa servers in "about:config"

**Comment by YaBoiMax** [comment]
Well I just deleted the dest folder and reran ./init.sh, started the docker-compose and at the beginning I had a problem like #13 yesterday, but I just let it run and wanted to get back to you today. But I just tried to login and now verything works for some reason. I have not changed anything from last time other than deleting the ./dest folder and rerunning the containers. If I ever encounter this problem I'll try and troubleshoot it immediatly so maybe we can figure out why this happens. For now I'll close this problem though
